src/euphorie/content/browser/country.py

```diff
@@ -101,3 +104,38 @@ class Tools(CountryView):
     def get_tools(self, sector_id):
         sector = self.context.get(sector_id)
         return getSurveys(sector)
+
+    def download_csv(self):
+        fieldnames = [
+            "Sector",
+            "Tool Title",
+            "Version Title",
+            "URL",
+            "published",
+            "obsolete",
+        ]
+        buffer = StringIO()
+        writer = csv.DictWriter(buffer, fieldnames=fieldnames, dialect="bilbomatica")
```

Comment:
why the dialect `bilbomatica`?‌ the default is `excel` which would be what users most often expect and what the ticket description asks for.
If there is a reason, please leave a comment in code.

Reply:
I went for consistency with [existing code](https://github.com/euphorie/Euphorie/blob/country-tools-csv/src/euphorie/content/utils.py#L27), but thinking about it again, you're probably right and the default is better if nothing specific was requested.